sockeye/encoder.py

```diff
@@ -85,14 +87,17 @@ class Encoder:
     Generic encoder interface.
     """
 
-    def encode(self, data: mx.sym.Symbol, data_length: mx.sym.Symbol, seq_len: int) -> mx.sym.Symbol:
+    def encode(self,
+               data: mx.sym.Symbol,
+               data_length: mx.sym.Symbol,
+               seq_len: int) -> (mx.sym.Symbol, mx.sym.Symbol, int):
```

Comment:
is mypy happy with this typing or should we use:
`Tuple[mx.sym.Symbol, mx.sym.Symbol, int]` ?

(see also below)